libs/test/MobileSyncTest/src/com/salesforce/androidsdk/mobilesync/config/SyncsConfigTest.java

```diff
@@ -177,19 +176,19 @@ public void testRefreshSyncDownFromConfig()  throws JSONException {
     }
 
     @Test
-    public void testLayoutSyncDownFromConfig()  throws JSONException {
+    public void testLayoutSyncDownFromConfig() throws JSONException {
```

Comment:
We should probably update:
- the schema https://github.com/forcedotcom/SalesforceMobileSDK-Package/blob/master/shared/syncs.schema.json#L124
- the example syncs configs: https://github.com/forcedotcom/SalesforceMobileSDK-Package/blob/master/shared/example.usersyncs.json#L55 and https://github.com/forcedotcom/SalesforceMobileSDK-Package/blob/master/shared/example.usersyncs.json#L55 and https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/master/libs/MobileSync/MobileSyncTestApp/usersyncs.json#L55

Reply:
Will do. I had forgotten that we had added these.